desktop/libs/notebook/src/notebook/templates/editor_components.mako

```diff
@@ -589,6 +589,22 @@ from notebook.conf import ENABLE_QUERY_BUILDER, ENABLE_QUERY_SCHEDULING, ENABLE_
     <div class="tab-pane ${ 'active' if not has_optimizer() else '' }" id="sessionsTab">
       <div class="row-fluid">
         <div class="span12" data-bind="template: { name: 'notebook-session-config-template', data: $root }"></div>
+            <!-- this is the session reloading spinner -->
```

Comment:
Somehow need to grab the session and use its attribute, similar to this maybe?

  <!-- ko hueSpinner: { spin: session.restarting(), center: true } --><!-- /ko -->